.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Build manage-frontend
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  CI:
+    if: >-
+      (github.actor != 'dependabot[bot]') &&
+        (github.event.pull_request.head.repo.owner.login == 'guardian' ||
+          github.event_name == 'push')
+    # Required by actions-assume-aws-role
+    permissions:
+      id-token: write
+      contents: read
+    name: manage-frontend build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Env
+        run: env
+
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
Let's use the [latest version](https://github.com/actions/checkout/blob/main/CHANGELOG.md) of this action, v4.